webapp/channels/src/selectors/rhs.ts

```diff
@@ -123,6 +124,15 @@ export function getSearchTerms(state: GlobalState): string {
     return state.views.rhs.searchTerms;
 }
 
+// getSearchTeam returns the team ID that the search is currently scoped to, or the current team if no team was specified.
+export function getSearchTeam(state: GlobalState): string {
+    const searchTeam = state.views.rhs.searchTeam;
+    if (searchTeam === undefined || searchTeam === null) {
+        return getCurrentTeamId(state);
+    }
+    return searchTeam;
```

Comment:
Functionally the same, but 0/5 just fine as-is
```suggestion
    return state.views.rhs.searchTeam ?? getCurrentTeamId(state);
```

Reply:
TIL! I was sure that `??` would consider an empty string as false